src/Playroom/CodeEditor/CodeEditor.tsx

```diff
@@ -149,7 +149,7 @@ export const CodeEditor = ({ code, onChange, previewCode, hints }: Props) => {
     return () => {
       window.removeEventListener('keydown', handleKeyDown);
     };
-  }, [dispatch, setCursorPosition]);
+  }, [dispatch]);
```

Comment:
Looks like we should have linting for this. Should look to add this at some stage.